examples/examples.py

```diff
@@ -51,22 +52,27 @@
         "scenario": "example_01d",
         "study_case": "zonal_case",
     },
-    "large_2019_eom": {"scenario": "example_02", "study_case": "base_case_2019"},
+    # example_01e is used in the tutorial notebook #3: Custom unit and bidding strategy example
+    # example_01f is used in the tutorial notebook #5: Market configuration comparison example
+    # example_01g is used in the tutorial notebook #6: Advanced order types example
+    #
+    # small examples with DRL enabled to show the learning process and configuration
+    "small_learning_1": {"scenario": "example_02a", "study_case": "base"},
+    "small_learning_2": {"scenario": "example_02b", "study_case": "base"},
```

Comment:
I would suggest sorting the Rl exampels into the ones representing the cass in your study @nick-harder and then other RL exampels.

Reply:
but isn't this already the case? or what do you mean?

Reply:
I have now added a clear reference to the cases from the publication in the rst description

Reply:
pushed my thought